src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react'
+import { patchArticleVotes } from "../utils"
+
+export default function ArticleVotes({articleVotes, articleId}) {
+    const [ votes, setVotes ] = useState(articleVotes)
+    const [ error, setError ] = useState(null)
+    const [ clicked, setClicked ] = useState(null)
+
+    function handleArticleVote(event, id, vote) {
+      if(clicked){
+        vote = vote*2
```

Comment:
I see your thinking here, but its not a great user experience currently to have the upvote increase by one, then if you press downvote it decreases by 2. It would be great if you could add functionality to instead have a neutral ground. For example upvote means votes are increased by 1, you can then unclick upvote which decreases votes by 1, then click downvote to decrease by 1.